src/fidesops/schemas/dataset.py

```diff
@@ -62,14 +62,18 @@ class FidesCollectionKey(ConstrainedStr):
     def validate(cls, value: str) -> str:
 
         values = value.split(".")
-        if (
-            len(values) == 2
-            and FidesKey.validate(values[0])
-            and FidesKey.validate(values[0])
-        ):
-            return value
-
-        raise FidesValidationError(
+        try:
+            if (
+                len(values) == 2
+                and FidesOpsKey.validate(values[0])
+                and FidesOpsKey.validate(values[0])
+            ):
+                return value
+        except ValueError:
+            raise ValueError(
+                "FidesCollection must be specified in the form 'FidesKey.FidesKey'"
```

Comment:
There's a bug here, you're catching the ValueError and re-raising with a message that isn't accurate, I'd do something like this:

```python
    @classmethod
    def validate(cls, value: str) -> str:
        """
        Overrides validation to check FidesCollectionKey format, and that both the dataset
        and collection names have the FidesKey format.
        """
        values = value.split(".")
        if len(values) == 2:
            FidesOpsKey.validate(values[0])
            FidesOpsKey.validate(values[1])
            return value
        raise ValueError(
            "FidesCollection must be specified in the form 'FidesKey.FidesKey'"
        )

```

Reply:
ohhh good point, the only case that pertains to this error message is when `len(values) != 2`